scripts/lib.sh

```diff
@@ -37,6 +37,11 @@ function await_server_on_port() {
 # Function for killing processes stored in FM_PID_FILE
 function kill_minimint_processes {
   kill $(cat $FM_PID_FILE | sed '1!G;h;$!d') #sed reverses the order here
-  pkill "ln_gateway";
+  pkill "ln_gateway" || true;
```

Comment:
This was breaking CI because the gateway plugin is no longer run by the rust tests. So we just kill the gateway if it's running and don't fail if it isn't running.